src/lib.rs

```diff
@@ -1396,10 +1409,18 @@ pub extern "C" fn zendoo_free_poseidon_hash(
 pub extern "C" fn zendoo_new_ginger_mht(
     height: usize,
     processing_step: usize,
+    ret_code: &mut CctpErrorCode,
```

Comment:
Please recrusively update accordingly all the places this function is used, since you changed the interface (it is called by the ZendooGingerMerkleTree constructor in zendoo_mc.h, so then look for its usages in the c++/python tests and update accordingly)

Reply:
Done